.github/workflows/native-oci-generative-ai.yml

```diff
@@ -0,0 +1,31 @@
+name: native-oci-generative-ai
+on:
+  push:
+    paths:
+      - 'native-oci-generative-ai/**'
+      - '.github/workflows/native-oci-generative-ai.yml'
+  pull_request:
+    paths:
+      - 'native-oci-generative-ai/**'
+      - '.github/workflows/native-oci-generative-ai.yml'
+  schedule:
+    - cron: "0 0 1 * *" # run every month
+  workflow_dispatch:
+permissions:
+  contents: read
+jobs:
+  run:
+    name: Run 'native-oci-generative-ai'
+    runs-on: ubuntu-latest
+    timeout-minutes: 20
+    steps:
+      - uses: actions/checkout@v4
+      - uses: graalvm/setup-graalvm@v1
+        with:
+          java-version: '21.0.2'
```

Comment:
Can we also test this against `latest-ea` to ensure updates don't break the demo? Also, the latest JDK 21 release is `21.0.3`.